inst/@sym/besselyn.m

```diff
@@ -0,0 +1,62 @@
+%% Copyright (C) 2016 Utkarsh Gautam 
+%%
+%% This file is part of OctSymPy.
+%%
+%% OctSymPy is free software; you can redistribute it and/or modify
+%% it under the terms of the GNU General Public License as published
+%% by the Free Software Foundation; either version 3 of the License,
+%% or (at your option) any later version.
+%%
+%% This software is distributed in the hope that it will be useful,
+%% but WITHOUT ANY WARRANTY; without even the implied warranty
+%% of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See
+%% the GNU General Public License for more details.
+%%
+%% You should have received a copy of the GNU General Public
+%% License along with this software; see the file COPYING.
+%% If not, see <http://www.gnu.org/licenses/>.
+
+%% -*- texinfo -*-
+%% @documentencoding UTF-8
+%% @deftypefn  {Function File} {@var{Y} =} besselyn (@var{alpha}, @var{x})
+%% Symbolic Spherical Bessel function of the second kind.
+%%
+%% Example:
+%% @example
+%% @group
+%% syms n x
+%% A = besselyn(n, x)
+%%   @result{} A = (sym) yn(n, x)
+%% diff(A)
+%%   @result{} ans = (sym)
+%%   
+%%                   (n + 1)⋅yn(n, x)
+%%    yn(n - 1, x) - ────────────────
+%%                          x 
```

Comment:
This is kind of picky, but please indent the `yn(n - 1, x) - ....` bit to line up under the `u` of `@result{}`